docs/howto/danceable-lyrics-input-tool/README.md

```diff
@@ -0,0 +1,485 @@
+# Danceable Lyrics
+In this guide, we use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create tool that connects directly to a set of data files and determines the most "danceable" songs containing a set of lyrics. We will be utilizing [Polars](https://www.pola.rs/) for fast data processing.
+
+# Download Input Data
+Download and extract the required dataset files. You will reference them later on in this guide. We provide to sets of this data -- the full and a truncated, smaller version.
```

Comment:
TW review:

Download and extract the required dataset files. You will reference them later in this guide. We provide 2 datasets—the full dataset and a smaller, truncated version.